akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -976,6 +976,33 @@ final case class `X-Forwarded-For`(addresses: immutable.Seq[RemoteAddress]) exte
   def getAddresses: Iterable[jm.RemoteAddress] = addresses.asJava
 }
 
+object `X-Forwarded-Host` extends ModeledCompanion[`X-Forwarded-Host`] {
+  implicit val addressRenderer = {
+    implicit val singleAddressRenderer = RemoteAddress.renderWithoutPort
+    Renderer.defaultSeqRenderer[RemoteAddress] // cache
```

Comment:
`X-Forwarded-Host` only holds one address, and this creates a renderer for a `Seq[RemoteAddress]`. There's no test to validate, but it looks like you want this:

```
implicit val singleAddressRenderer = RemoteAddress.renderWithoutPort
```

and the import in the `case class` should be updated accordingly.

Reply:
Yes, you are right. I will change this.